epoxy-adapter/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
   compile "com.android.support:recyclerview-v7:24.2.1"
   compile "com.android.support:design:24.2.1"
   compile "com.android.support:appcompat-v7:24.2.1"
-  compile project(":epoxy-model")
+  compile project(':epoxy-annotations')
```

Comment:
I also renamed the model module to `annotations`, since all it has is annotations now that the EpoxyModel was moved to `epoxy-adapter` module